.github/workflows/build-lint-push-containers.yml

```diff
@@ -102,16 +147,26 @@ jobs:
     needs: container-build-push
     runs-on: ubuntu-latest
     steps:
-      - name: Get latest commit info
+      - name: Get latest commit info (latest)
         if: github.event_name == 'push'
         run: |
           LATEST_COMMIT_HASH=$(echo ${{ github.event.after }} | cut -b -7)
           echo "LATEST_COMMIT_HASH=${LATEST_COMMIT_HASH}" >> $GITHUB_ENV
-      - name: Dispatch event for latest
-        if: github.event_name == 'push'
+
+      - name: Dispatch event (latest)
+        if: github.event_name == 'push' && ${{ env. PROWLER_VERSION_MAJOR }} == '3'
```

Comment:
This is gonna be available only for version 3?

Reply:
At least for now, we haven't tested it with v4 _yet_.